common/build/build-common/api-extractor-common.json

```diff
@@ -111,17 +104,26 @@
     "extractorMessageReporting": {
       "default": {
         "logLevel": "error"
-        // "addToApiReviewFile": false
       },
+      // A documentation comment should contain at most one release tag.
+      "ae-extra-release-tag": {
+        "logLevel": "warning"
+      },
+      // Reported when an exported API refers to another declaration that is not exported.
       "ae-forgotten-export": {
         "logLevel": "error"
       },
-      "ae-extra-release-tag": {
-        "logLevel": "warning"
+      // Disabled. We don't require that internal members be prefixed with an underscore.
+      "ae-internal-missing-underscore": {
+        "logLevel": "none",
+        "addToApiReportFile": false
```

Comment:
addToApiReportFile defaults to false, so why set it here?

Reply:
The comment that I removed was actually misleading (I believe it was copied from another of the api-extractor messaging config properties). The default value for `addToApiReportFile` is true, not false. I would expect this to not be needed since logLevel is set to `none`, but I was seeing a warning getting reported without explicitly setting this.

I'm doing a deeper investigation / config cleanup on a separate branch that will hopefully remove the need for this, but I figured I would get the quick fix in first.